src/messen.ts

```diff
@@ -136,7 +136,7 @@ export class Messen {
   }
 
   listen(): void {
-    this.api.listen((err, ev) => {
+    this.api.listenMqtt((err, ev) => {
```

Comment:
@ethan-ou can you add this type to `src/types/facebook-chat-api.d.ts` in the `API` class below where `listen` is defined?

```javascript
listenMqtt(
      callback: (
        err: Facebook.FacebookError | undefined,
        event: Facebook.APIEvent,
      ) => void,
```

Reply:
Completely forgot I was working in Typescript! All done now.